jujugui/static/gui/src/app/app.js

```diff
@@ -2113,7 +2112,6 @@ YUI.add('juju-gui', function(Y) {
    */
     _ensureControllerConnection: function(state, next) {
       if (
-        state.root !== 'logout' &&
```

Comment:
hmm, why isn't this necessary? I know we kind of wrote this part assuming it was, just curious.